container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -318,10 +319,24 @@ function run_git_tempest {
     tempest init openshift
     pushd $TEMPEST_DIR
 
+    # We're running cleanup only under certain circumstances
+    if [[ ${TEMPEST_DEBUG_MODE} == false ]] && [[ ${TEMPEST_CLEANUP} == true ]]; then
+        # generate a simple tempest.conf so that we can run --init-saved-state
+        discover-tempest-config
```

Comment:
Won't this still keep the cirros image hanging?

```
(.venv) stack@lpiwowar:~/lpiwowar/tempest$ openstack image list

(.venv) stack@lpiwowar:~/lpiwowar/tempest$ discover-tempest-config
...
2024-09-18 12:17:39.957 3158161 INFO tempest.lib.common.rest_client [req-4f2727c6-d64e-4653-b029-28994322d87b req-4f2727c6-d64e-4653-b029-28994322d87b ] Request (main): 200 GET http://10.0.144.252/image/v2/images 0.011s
2024-09-18 12:17:39.957 3158161 INFO config_tempest.constants [-] Creating image 'cirros-0.6.2-x86_64-disk.img'
2024-09-18 12:17:39.957 3158161 INFO config_tempest.constants [-] Downloading 'https://download.cirros-cloud.net/0.6.2/cirros-0.6.2-x86_64-disk.img' and saving as '/opt/stack/lpiwowar/tempest/etc/cirros-0.6.2-x86_64-disk.img'
2024-09-18 12:17:59.038 3158161 INFO config_tempest.constants [-] Uploading image 'cirros-0.6.2-x86_64-disk.img' from '/opt/stack/lpiwowar/tempest/etc/cirros-0.6.2-x86_64-disk.img'
2024-09-18 12:17:59.274 3158161 INFO tempest.lib.common.rest_client [req-8a28c07e-8e5c-4d07-afb7-ba813f53effb req-8a28c07e-8e5c-4d07-afb7-ba813f53effb ] Request (main): 201 POST http://10.0.144.252/image/v2/images 0.235s
2024-09-18 12:17:59.848 3158161 INFO tempest.lib.common.rest_client [req-87ff9abf-8d43-4b24-bd70-76da9f1d13cd req-87ff9abf-8d43-4b24-bd70-76da9f1d13cd ] Request (main): 204 PUT http://10.0.144.252/image/v2/images/0b10b998-e455-4829-90ae-e70aef8cc6e0/file 0.574s
...
(.venv) stack@lpiwowar:~/lpiwowar/tempest$ openstack image list
+--------------------------------------+----------------------------------+--------+
| ID                                   | Name                             | Status |
+--------------------------------------+----------------------------------+--------+
| 0b10b998-e455-4829-90ae-e70aef8cc6e0 | cirros-0.6.2-x86_64-disk.img     | active |
| 4cbfd5fe-f715-4a83-b505-19262e1115e1 | cirros-0.6.2-x86_64-disk.img_alt | active |
+--------------------------------------+----------------------------------+--------+
```

Reply:
yup, we have 2 options then:
1. make discover-tempest-config not to create any images, flavors etc, just a very simple tempest.conf
2. execute 'openstack image delete cirros-0.6.2-x86_64-disk.img' and 'openstack image delete cirros-0.6.2-x86_64-disk.img_alt' commands

I'll check the options and get back to you

Reply:
Ok, got it. I agree it seems using the openstack CLI is the best solution right now.